docs/content/quickstart/datastream-api-package-guidance.md

```diff
@@ -0,0 +1,242 @@
+# DataStream api package guidance
+
+This guide provides a simple pom example of mysql cdc DataStream api
+
+## frame version
+
+flink 1.13.6  flink mysql cdc 2.4.1
+
+## pom example
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.ververica</groupId>
+    <artifactId>FlinkCDCTest</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>1.8</java.version>
+        <scala.binary.version>2.12</scala.binary.version>
+        <maven.compiler.source>${java.version}</maven.compiler.source>
+        <maven.compiler.target>${java.version}</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <!-- Enforce single fork execution due to heavy mini cluster use in the tests -->
+        <flink.forkCount>1</flink.forkCount>
+        <flink.reuseForks>true</flink.reuseForks>
+
+        <!-- dependencies versions -->
+        <flink.version>1.13.6</flink.version>
```

Comment:
Thanks @gtk96 for the continuous work, could we add other flink version example like 1.14/1.15/1.16/1.17 , we can simply list them in note way? the different should include `flink-shaded-guava`.